user_data_model/Makefile

```diff
@@ -67,22 +67,10 @@ build-pdm:
 	$(PSYNC_HOME)/modules/navigation/navigation.idl \
 	$(PSYNC_HOME)/modules/control/control.idl \
 	idl/user_data_model.idl > pdm/version.txt
+
 # show the results
 	cat pdm/error.log
 
-# install our example data model into the system
```

Comment:
I feel like these should stay. Shea updated them earlier to install to the right location. 

Alternatively we can move the install commands to the README and other documentation.